model/schema/core.yaml

```diff
@@ -517,16 +521,34 @@ slots:
     range: string
 
   start:
-    description: The start of the feature in positive 1-based integer coordinates
+    description: >-
+      The start of the feature in positive 1-based integer coordinates relative
+      to the reference landmark.
+    domain: LocationAssociation
     range: integer
 
   end:
-    description: The end of the feature in positive 1-based integer coordinates
+    description: >-
+      The end of the feature in positive 1-based integer coordinates relative
+      to the reference landmark.
+    domain: LocationAssociation
     range: integer
 
+  # QUESTION: How to specify strandedness for genomic location.
+  strand:
```

Comment:
https://github.com/biolink/biolink-model/blob/db44be0c49939229c28cbb71a715127941e0ce0b/biolink-model.yaml#L5711C1-L5711C1

https://github.com/biolink/biolink-model/blob/db44be0c49939229c28cbb71a715127941e0ce0b/biolink-model.yaml#L11264 

?

Reply:
Perfect. Thank you!